### PR TITLE
Fix flint skipping index syntax issues

### PR DESCRIPTION
**Pull request by YANG-DB**
### Description
update vpc flow logs & cloud trail flint related integration content changes

#### CloudTrail

> Minor fix for the Amazon Log Integration for Flint Version 1.1.0
>   Fix skipping index related [issue](https://github.com/opensearch-project/opensearch-spark/issues/353)
>   Update table creation statement according to [Athena DDL Statement](https://docs.aws.amazon.com/athena/latest/ug/cloudtrail-logs.html#create-cloudtrail-table-understanding) 
> See related [Athena S3 setup tutorial](https://aws.amazon.com/blogs/big-data/aws-cloudtrail-and-amazon-athena-dive-deep-to-analyze-security-compliance-and-operational-activity/)

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json
Comment (RyanL1997) [inline server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json:97]: Just for my knowledge, what is the record referencing to?

File: server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json
Comment (amsiglan) [inline server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json:32]: Same label as above, will this be confusing to the user?

File: server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_table_parquet_vpc-1.0.0.sql
Comment (RyanL1997) [inline server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_table_parquet_vpc-1.0.0.sql:61]: Nice. Unrelated to this change, but I do notice that there is an empty file called "create_mv_vpc-1.0.0.sql" in the vpc asset directory. Should we also remove that?

**Review by RyanL1997 — COMMENTED** [review]
In generally LGTM. Just left some minor thoughts.

File: server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json
Comment (YANG-DB) [reply server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json:97]: extended format for multiple records shown [here](https://aws.amazon.com/blogs/big-data/aws-cloudtrail-and-amazon-athena-dive-deep-to-analyze-security-compliance-and-operational-activity/)

Comment (Swiddis) [reply server/adaptors/integrations/__data__/repository/aws_cloudtrail/aws_cloudtrail-1.0.0.json:97]: Do I understand right that we only have acceleration for single-record, and multi-record has queries without acceleration? LGTM technically, but I wonder how common multi-record is compared to single-record/what the impact of that is.

File: server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_mv_cloud-trail-records-1.0.0.sql
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_mv_cloud-trail-records-1.0.0.sql:48]: Are there issues with having the table alias default to `myTable`? Looks like it probably won't cause issues since it's just a virtual table but I'm not sure where the virtual table actually gets stored or if this should be more descriptively named.

**Review by Swiddis — CHANGES_REQUESTED** [review]
I know we discussed at one point enabling the create table query by default so it doesn't cause an error if running with only queries selected and no other flows -- would now be a good time to implement that? OTOH I don't want to rock the boat since it's not in-scope for the immediate issue.

Aside from that, there's one unescaped field still -- did a full diff with the known-working version we made yesterday and that's the only delta so I can approve after that.

File: server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_skipping_index-1.0.0.sql
Comment (YANG-DB) [reply server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_skipping_index-1.0.0.sql:9]: good catch - thanks

**Review by Swiddis — APPROVED** [review]

**Review by Swiddis — APPROVED** [review]

File: server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_skipping_index-1.0.0.sql
Comment (ps48) [inline server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_skipping_index-1.0.0.sql:9]: Are we removing the field `request_processing_time MIN_MAX,`?

File: server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_table_cloud-trail-records-1.0.0.sql
Comment (ps48) [inline server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_table_cloud-trail-records-1.0.0.sql:66]: Question? Should this be `multiline='true'` .

File: server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_table_cloud-trail-1.0.0.sql
Comment (ps48) [inline server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_table_cloud-trail-1.0.0.sql:63]: Is multiline part case sensitive. I've usually seen commands with `multiLine` L captialised.

Comment (ps48) [reply server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_table_cloud-trail-1.0.0.sql:63]:
```
PATH='{s3_bucket_location}',
recursivefilelookup='true',
multiLine='true'
```

Comment (YANG-DB) [reply server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_table_cloud-trail-1.0.0.sql:63]: I think both `multiline` and `multiLine` are be accepted

File: server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_skipping_index-1.0.0.sql
Comment (Swiddis) [reply server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_skipping_index-1.0.0.sql:9]: It doesn't exist in the data schema, so it wasn't supposed to be there anyways

**Review by RyanL1997 — APPROVED** [review]

**Review by ps48 — APPROVED** [review]

**Pull request merged**
